ui/cypress/e2e/standards/list.cy.js

```diff
@@ -13,8 +15,11 @@ describe('Standards Listing Index', () => {
   describe('Search', () => {
     it('Can search by standard matching', () => {
       cy.visit('/standards');
+      cy.injectAxe();
       cy.doSearch('allergies');
       cy.get('#browse-results li').not('have.length', 0);
+      // cy.checkA11y(context, options, violationCallback, skipFailures);
```

Comment:
why this?

Reply:
It's just a reference for me/future me. I can remove it if it feels bad to keep it

Reply:
I'm just not sure we comment signatures anywhere else? I'd rather remove it